### ISSUE TITLE
Duplicate WT_SESSION and WT_PROFILE_ID in WSLENV env var

### ISSUE THREAD

**Issue by abarani** [microsoft/terminal#7130]
# Environment

```none
Windows build number: 10.0.19041.388
Windows Terminal version (if applicable): 1.1.2021.0
```

# Steps to reproduce

- Open Windows Terminal
- `setx WSLENV "${env:WSLENV}:TEST/p"`
- Close and reopen Windows Terminal
- `echo $env:WSLENV`

# Expected behavior

    WT_SESSION:WT_PROFILE_ID:TEST/p

# Actual behavior

    WT_SESSION:WT_SESSION::WT_PROFILE_ID:TEST/p:WT_PROFILE_ID

---

I think the right thing to do here is to split `WSLENV` by `:` and check those vars are already set instead of just simply prepending and appending a string.

Probable related PRs:
- https://github.com/microsoft/terminal/pull/4157
- https://github.com/microsoft/terminal/pull/4852

**Comment by DHowett** [comment]
Good catch! There's an inherent danger in using `setx` to immortalize the contents of an environment variable at runtime, because even `TEST/p` is going to get duplicated if you keep doing that ...

I'm not sure it harms anything other than aesthetics, but I'll tag this one up as a code health issue.

**Comment by huiyooumich** [comment]
Hello, this is my first time making a contribution here! I would like to work on this.

**Comment by zadjii-msft** [comment]
@Dannihu01 awesome, go for it! I don't think you'd be stepping on anyone's toes here. I'm pretty sure `ConptyConnection::_LaunchAttachedClient` in `ConptyConnection.cpp` is where we're setting up the environment variables. Let us know if you need any pointers ☺️

**Comment by WanderLanz** [comment]
This seems to have been solved* then by loading an extra settings.json map "environment", ConptyConnection::Initialize at line ~268 in ConptyConnection.cpp. 

I haven't touched c++ in a while but it seems to be valid, and it's been there since before v1, yet it's not in the settings.json schema and doesn't seem to be working properly as far as I've tested. 
Is "environment" meant to be in the settings? 

Otherwise, it should probably be explicitly discouraged to use "SETX" on WSLENV by appending (e.g. setx WSLENV "%WSLENV%:WT_SETTINGS/p").
Note that {setx WSLENV "WT_SETTINGS/p"} will result in WSLENV="WT_SESSION:WT_SETTINGS/p:WT_PROFILE_ID" as intended because ConptyConnection.cpp prepends & appends rather than overrides the existing WSLENV.

...
This took an embarrassing long time for me to figure out :P, I was going to mess with ways to preview images in WT in-window by hacking/setting the background of a temp profile in a split pane, since that seemed like a good idea

**Comment by VISHESH0932** [comment]
@abarani , I would like to work on this issue.

**Comment by Weichenleeeee123** [comment]
Hi , I'm a newcomer here and i would like to take this task if there's no one had taken it.

**Comment by DHowett** [comment]
@Weichenleeeee123 thanks for working on this bug!
